recommended_ruleset.js

```diff
@@ -133,7 +133,7 @@ module.exports = {
         "no-for-in": true,
         "no-function-expression": true,
         "no-inferrable-types": false, // turn no-inferrable-types off in order to make the code consistent in its use of type decorations
-        "no-multiline-string": false,
+        "no-multiline-string": true, // multiline-strings often introduce unnecessary whitespace into the string literals
```

Comment:
Looks like some unrelated changes on this file made it in. Could you please undo/revert changes to `recommended_ruleset.js` and `package-lock.json`?

Separately, I think we should start checking in `package-lock.json`, so I've filed #462 to do that.